photo/models.py

```diff
@@ -1,3 +1,101 @@
+from datetime import timezone
+
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
 from django.db import models
 
+
 # Create your models here.
+class User(AbstractBaseUser, PermissionsMixin):
+    USERNAME_FIELD = "email"
+
+    email = models.CharField(max_length=255, primary_key=True)
+    name_first = models.CharField(max_length=255, blank=True, null=True)
+    name_last = models.CharField(max_length=255, blank=True, null=True)
+    profile_picture = models.ForeignKey(
+        "Picture",
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        db_column="picture_id",
+    )
+    profile_picture_updated_at = models.DateTimeField(blank=True, null=True)
+    user_handle = models.CharField(max_length=255, blank=True, null=True)
+
+
+class Picture(models.Model):
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        db_column="user_id",
+    )
+    picture_url = models.CharField(max_length=255, primary_key=True)
+    likes = models.ManyToManyField(User, blank=True, null=True)
+
+
+class PictureComment(models.Model):
```

Comment:
we should add a UUID for this, we can have duplicate comments if the users want to